target/linux/brcm63xx/dts/av4202n.dts

```diff
@@ -71,20 +71,4 @@
 	status = "ok";
 
 	linux,part-probe = "bcm63xxpart";
-
-	cfe@0 {
-		label = "CFE";
-		reg = <0x000000 0x020000>;
-		read-only;
-	};
-
-	linux@20000 {
-		label = "linux";
-		reg = <0x010000 0xfe0000>;
```

Comment:
Does it work if you only fix the obviously wrong start offset to ```0x020000```? I want to remove the use of the automated detection of the image offset as much as possible, so removing the partitions would be a step back.